packages/common/src/content/index.ts

```diff
@@ -490,95 +269,61 @@ export function itemToContent(item: IItem): IHubContent {
-  delete content.itemCategories;
-
-  // overwrite or add enrichments from Hub API
-  const { id: hubId, attributes } = dataset;
+  // extract enrichments from attributes
   const {
-    // common enrichments
+    // item enrichments
     errors,
     boundary,
-    extent,
     metadata,
-    modified,
-    modifiedProvenance,
     slug,
-    searchDescription,
     groupIds,
+    orgId,
+    orgName,
+    organization,
+    orgExtent,
     // map and feature server enrichments
     server,
     layers,
-    // NOTE: the Hub API also returns the following server properties
-    // but we should be able to get them from the above server object
-    // currentVersion, capabilities, tileInfo, serviceSpatialReference
-    // maxRecordCount, supportedQueryFormats, etc
-    // feature and raster layer enrichments
     layer,
     recordCount,
     statistics,
-    // NOTE: the Hub API also returns the following layer properties
-    // but we should be able to get them from the above layer object
-    // supportedQueryFormats, supportsAdvancedQueries, advancedQueryCapabilities, useStandardizedQueries
-    // geometryType, objectIdField, displayField, fields,
-    // org properties?
-    orgId,
-    orgName,
-    organization,
-    orgExtent,
-    // NOTE: for layers and tables the Hub API returns the layer type
-    // ("Feature Layer", "Table", "Raster Layer") instead of the item type
-    type,
-  } = attributes;
+    // additional attributes needed
+    searchDescription,
+  } = dataset.attributes;
 
-  // NOTE: we could throw or return if there are errors
-  // to prevent type errors trying to read properties below
-  content.errors = errors;
+  // get the layerId from the layer
+  const layerId = layer && layer.id;
 
-  // common enrichments
-  content.boundary = boundary;
-  if (!isBBox(item.extent) && extent && isBBox(extent.coordinates)) {
-    // we fall back to the extent derived by the API
-    // which prefers layer or service extents and ultimately
-    // falls back to the org's extent
-    content.extent = extent.coordinates;
-  }
-  // setting this to null signals to enrichMetadata to skip this
-  content.metadata = metadata || null;
-  if (content.modified !== modified) {
-    // capture the enriched modified date
-    // NOTE: the item modified date is still available on content.item.modified
-    content.modified = modified;
-    content.updatedDate = new Date(modified);
-    content.updatedDateSource = modifiedProvenance;
-  }
-  content.slug = slug;
+  // re-assemble the org as an enrichment
+  const org = orgId && {
+    id: orgId,
+    name: orgName || organization,
+    extent: orgExtent,
+  };
+
+  // compose a content out of the above
+  const content = composeContent(item, {
+    layerId,
+    slug,
+    errors,
+    // setting this to null signals to enrichMetadata to skip this
+    metadata: metadata || null,
+    groupIds,
+    org,
+    server,
+    layers,
+    recordCount,
+    boundary,
+    statistics,
+  });
```

Comment:
here's the payoff, `datasetToContent()` is now using the exact same logic as `itemToContent()`, it's just passing in more enrichments that we got form the API